.github/workflows/run_integration_tests.yml

```diff
@@ -66,10 +66,35 @@ jobs:
         run: |
           source $HOME/BYOND/byond/bin/byondsetup
           bash tools/ci/run_server.sh ${{ inputs.map }}
-      - name: Upload Code Coverage
+      - name: Upload Code Coverage Full
         uses: codecov/codecov-action@v3
         with:
-          files: ci_test/auxtools_coverage.xml
+          files: ci_test/coverage/full.xml
+          flags: full
+      - name: Upload Code Coverage Init
```

Comment:
As a nit, we should do all this after uploading screenshot tests--they already take a while.